addie/processing/idl/populate_master_table.py

```diff
@@ -24,8 +24,7 @@ def run(self):
             self.read_auto_sum_file()
             self.populate_table()
         except IOError:
-            _error_box = QMessageBox.warning(self.parent, "File does not exist!", "Check your folder!         ")
-#            _error_box.show()
+            QMessageBox.warning(self.parent, "File does not exist!", "Check your folder!         ")
```

Comment:
```suggestion
            QMessageBox.warning(self.parent, "File does not exist!", "Check your folder!")
```

Reply:
Well, the white spaces are necessary as they allow to display the entire message within the MessageBox, otherwise, this one is truncated.